src/common/json_helper.h

```diff
@@ -347,8 +347,8 @@ inline bool json_decode(const JsonObject &in, dsn::ranger::access_type &t)
 {
     dverify(in.IsUint64());
     auto ans = in.GetUint64();
-    auto act_min = static_cast<uint64_t>(dsn::ranger::access_type_min);
-    auto act_max = static_cast<uint64_t>(dsn::ranger::access_type_max);
+    auto act_min = static_cast<uint64_t>(dsn::ranger::kAccessTypeMin);
```

Comment:
Maybe use 'empty'/'none' and 'full'/'all' instead of 'min' and 'max'? The first sight of min and max looks like the value of access_type, i.e. kInvalid and kControl.